pkg/vulnsrc/redhat-oval/redhat-oval.go

```diff
@@ -342,6 +342,15 @@ func parseDefinitions(advisories []redhatOVAL, tests map[string]rpmInfoTest, uni
 
 			var cveEntries []CveEntry
 			for _, cve := range advisory.Metadata.Advisory.Cves {
+				// get details from NVD
+				details, err := vs.dbc.GetVulnerabilityDetail(cve.CveID)
```

Comment:
VulnSrc should be order-agnostic. It means NVD must be inserted before Red Hat OVAL. I'd suggest adding CveIDs here.
https://github.com/aquasecurity/trivy-db/blob/2bd1364579ec652f8f595c4a61595fd9575e8496/pkg/types/types.go

And check the rejected status of the related CVE-IDs as well here.
https://github.com/aquasecurity/trivy-db/blob/930461748b6307590f4718daeab37cec51fe78b3/pkg/vulnsrc/vulnerability/vulnerability.go#L178-L189